.browserslistrc

```diff
@@ -1,3 +1,5 @@
 # Browsers that we support
 
-chrome >= 58
+chrome >= 64
+safari >= 11
+firefox >= 58
```

Comment:
those for examples are browsers released Jan 2018.
But to be honest i would like to decide by canvas features.
Like which is the minimum set of browsers with advanced text metrics?
For the beta is not relevant we can readjust